reference/ros-api.md

```diff
@@ -359,6 +359,7 @@ for more information about the value of `robot_unique_name`).
 | [/supervisor/get\_self](supervisor.md#wb_supervisor_node_get_from_def)                                                | service       | webots\_ros::supervisor\_get\_self                                   | uint8 ask<br/>---<br/>uint64 node                                                                                                                                                                                                                                 |
 | [/supervisor/get\_from\_def](supervisor.md#wb_supervisor_node_get_from_def)                                           | service       | webots\_ros::supervisor\_get\_from\_def                              | string name<br/>---<br/>uint64 node                                                                                                                                                                                                                               |
 | [/supervisor/get\_from\_id](supervisor.md#wb_supervisor_node_get_from_def)                                            | service       | webots\_ros::supervisor\_get\_from\_id                               | int32 id<br/>---<br/>uint64 node                                                                                                                                                                                                                                  |
+| [/supervisor/get\_selected](supervisor.md#wb_supervisor_node_get_from_def)                                            | service       | webots\_ros::supervisor\_get\_selected                               | uint8 ask<br/>---<br/>uint64 node                                                                                                                                                                                                                                 |
```

Comment:
we already did some service minimization [here.](https://github.com/omichel/webots/pull/4950/files/07165f4c9d64358214dca96c24711d732c6b1cec#diff-f17989f4e1d50b96dd0b8e374b6bf211)